backend/frontend/templates/frontend/user/component_card.html

```diff
@@ -1,25 +1,144 @@
+                    </div>
+                    <div class="{{styles.bad_review_counter_chip}}">{{card.code_review_ny_competent_since_last_review_request}} 
+                    </div>
+                </div>
+            {% endif %}
+            {% if card.code_review_red_flag_since_last_review_request %}
+                <div class="flex" title="Number of competent code reviews since last review request">
+                    <div>
+                        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M7.57 20c-1.34 0-2.43-1.09-2.43-2.43c0-1.07 1.18-3.07 2.43-4.76C8.82 14.5 10 16.5 10 17.57A2.43 2.43 0 0 1 7.57 20M12 2A10 10 0 0 0 2 12c0 1.75.45 3.38 1.24 4.81c.16-.81.57-1.74 1.07-2.64A7.69 7.69 0 0 1 4 12a8 8 0 0 1 8-8a8 8 0 0 1 8 8a8 8 0 0 1-8 8c-.24 0-.47 0-.71-.04c-.47.74-1.15 1.32-1.95 1.67c.85.24 1.74.37 2.66.37a10 10 0 0 0 10-10A10 10 0 0 0 12 2m0 12c-.41 0-.81.04-1.19.12c.35.63.66 1.28.88 1.88H12c1.25 0 2.32.5 2.77 1.23l1.42-1.42C15.29 14.72 13.75 14 12 14m3.5-6c-.8 0-1.5.7-1.5 1.5s.7 1.5 1.5 1.5s1.5-.7 1.5-1.5S16.3 8 15.5 8M10 9.5C10 8.7 9.3 8 8.5 8S7 8.7 7 9.5S7.7 11 8.5 11s1.5-.7 1.5-1.5"/></svg>
+                    </div>                
+                    <div class="{{styles.bad_review_counter_chip}}">{{card.code_review_red_flag_since_last_review_request}} 
+                    </div>
+                </div>    
+            {% endif %}
+        </div>
+    {% endif %}
 
     <h3 
         class="{{ styles.heading3 }}">
         {{ card.title }}
     </h3>
 
+    {% if card.tag_names %}
+        <div class="flex gap-2 items-center flex-wrap">
+            {% for tag_name in card.tag_names %}
+                <span class="flex px-2 py-1 rounded-full border border-gray-300 text-xs">{{tag_name}}</span> 
+            {% endfor %}
+        </div>
+    {% endif %}
 
-    {% if card.request_user_can_start %}
+    {% if card.due_time %}
+        <div class="flex gap-1" title="Due time">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M12 20a7 7 0 0 1-7-7a7 7 0 0 1 7-7a7 7 0 0 1 7 7a7 7 0 0 1-7 7m0-16a9 9 0 0 0-9 9a9 9 0 0 0 9 9a9 9 0 0 0 9-9a9 9 0 0 0-9-9m.5 4H11v6l4.75 2.85l.75-1.23l-4-2.37V8M7.88 3.39L6.6 1.86L2 5.71l1.29 1.53l4.59-3.85M22 5.72l-4.6-3.86l-1.29 1.53l4.6 3.86L22 5.72Z"/></svg>
+            {% now "Y-m-d H:i:s" as current_datetime %}
+            {% if card.due_time|date:"U" > current_datetime|date:"U" %}
+                <p class="text-red-600">
+                    {{card.due_time|naturalday}} 
+                </p>
+                {% else %}
```

Comment:
Not seeing a difference here in this conditional check

Reply:
I'll double check